NLSY/nlsy_lib.R

```diff
@@ -1,17 +1,23 @@
 # This file contains functions for manipulating the NLSY97 dataset.
+library(Hmisc)
+library(tidyverse)
+library(scales)
+library(ggridges)
+library(mice)
+library(urbnthemes)
 
+set_urbn_defaults(style = "print")
 # Read the raw data
-nlsydf = readRDS(paste0(here::here(), "/NLSY/NLSY-college-finance.rds"))
+nlsydf = read_csv('NLSY/NLSY-college-finance_imp.csv') %>%
```

Comment:
I would prefer to keep using NLSY-college-finance.rds as the original NLSY dataset. We can have imputations stored in a separate file and merged onto the main file. Also, considering that we are planning to use the imputations only in R, perhaps we can store them in RDS files as reading from them is faster than from CSV files.